package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "collecticons-lib",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Minor is the second number. The third one refers to "patch"
```suggestion
  "version": "2.2.0",
```